test_utilities/bin/prepare_environment.sh

```diff
@@ -9,5 +9,5 @@
 set -ex
 
 dart pub global activate tuneup
-flutter channel master
+flutter channel stable
```

Comment:
intentional?

Reply:
Was trying to see if stable `dart analyze` would be less crashy, but it crashed in another way, so I reverted it.